Automattic/jetpack
# Admin bar: move the Reader admin bar item to the secondary group

Fixes https://github.com/Automattic/wp-calypso/issues/93534

- Related Calypso PR: https://github.com/Automattic/wp-calypso/pull/93749

## Proposed changes:

* Move the Reader admin bar item to the right side of the admin bar (the secondary group).
* Give the Help Center item a priority.

Simple before:
<img width="897" alt="Screen Shot 2024-08-21 at 4 45 12 PM" src="https://github.com/user-attachments/assets/d8b85509-371e-4ef9-8f31-f16c7840fb50">

Simple this branch:
<img width="896" alt="Screen Shot 2024-08-21 at 4 44 58 PM" src="https://github.com/user-attachments/assets/77aca063-d345-49c6-8158-69ceeaf48267">

Atomic before:
<img width="859" alt="Screen Shot 2024-08-21 at 4 53 14 PM" src="https://github.com/user-attachments/assets/813c8240-2302-4254-80c0-b3faa6c1bb71">

Atomic this branch:
<img width="861" alt="Screen Shot 2024-08-21 at 4 53 38 PM" src="https://github.com/user-attachments/assets/6ddafdc2-d4ad-4e06-aee1-8f893db8a652">


### Other information:


## Jetpack product discussion



https://github.com/Automattic/wp-calypso/issues/93534

## Does this pull request change what data or activity we track or use?



N/A

## Testing instructions:






* To test on WoA, go to the Plugins menu on a WordPress.com Atomic site. Click on the "Upload" button and follow the upgrade flow to be able to upload, install, and activate [the Jetpack Beta plugin](https://jetpack.com/download-jetpack-beta/). Once the plugin is active, go to Jetpack > Jetpack Beta, select your plugin, and enable the update/admin-menu-reader-placement branch.
* Also add define( 'JETPACK_MU_WPCOM_LOAD_VIA_BETA_PLUGIN', true ); to your wp-config.php file.
* Go to WP Admin.
* Verify that the Reader menu item is on the right, to the left of Help Center.
* Check that clicking the Reader item takes you to the Reader.
* Test on mobile screen sizes.
* Test on supported browsers.
* Test with different color schemes.
* Test with a RTL language.
* Repeat the testing steps with a Simple site. To test on Simple, run the following command on your sandbox:

bin/jetpack-downloader test jetpack-mu-wpcom-plugin update/admin-menu-reader-placement
